### PR TITLE
Add techmix data prep fun

### LINKED ISSUE #11: Once implemented techmix_sector plot in `pacta-dashboard-svelte` add a data_prep function here
Relates to PR: https://github.com/RMI-PACTA/pacta-dashboard-svelte/pull/38 and #10

### PR DESCRIPTION
**Pull request by MonikaFu**
Closes #11 

As discussed adding the data prep function for new techmix plot in the repo for the time being.

### CODE REVIEW

**Comment by jdhoffa** [comment]
Totally read this as "Add techmix data prep for fun" lol

File: main.R
[the remark this answers is not in the document]
Comment (jdhoffa) [reply main.R:1]: Thanks Antoine. 
This decision will need input from (at least) Nayra. 

(Note: I strongly suspect she will prefer we align P4I and P4B methodologies, but that's just a hunch)

[the remark this answers is not in the document]
Comment (jdhoffa) [reply main.R:1]: Decision to KEEP the plot as it currently appears in the interactive report (e.g. NO action necessary).

Decision is tracked here: https://dev.azure.com/RMI-PACTA/2DegreesInvesting/_workitems/edit/11662

**Review by jdhoffa — APPROVED** [review]

**Review by Antoine-Lalechere — APPROVED** [review]
As discussed here: https://dev.azure.com/RMI-PACTA/2DegreesInvesting/_workitems/edit/11662?src=WorkItemMention&src-action=artifact_link
It looks good to me

**Pull request merged**